tests/integration_test.go

```diff
@@ -0,0 +1,128 @@
+// +build integration
+
+package tests_test
```

Comment:
just "tests" based on dir conventions? unless this to work around some travis or +build issue

Reply:
yeah just because of dir conventions

Reply:
not sure I understand the question. Are you suggesting this should be in the "tests" pkg? Or asking if it's in the "tests_test" pkg b/c of dir conventions?

If the former, we've been using <pkg>_test pkg naming convention.
If the latter, yes :)

Reply:
Oh, hm. I forgot this is another golang thing. Either is acceptable and depends on whether the test is white or black box. I think here it's definitely black box so _test is appropriate.